ludwig/hyperopt/sampling.py

```diff
@@ -432,4 +130,166 @@ def get_build_hyperopt_sampler(strategy_type):
     return get_from_registry(strategy_type, sampler_registry)
 
 
-sampler_registry = {"grid": GridSampler, "random": RandomSampler, "pysot": PySOTSampler, "ray": RayTuneSampler}
+sampler_registry = {"ray": RayTuneSampler}
+
+
+# TODO: split to separate module?
```

Comment:
+1 to moving this to a separate file, perhaps ludwig/hyperopt/search_algos.py

Reply:
Addressed by PR https://github.com/ludwig-ai/ludwig/pull/2082